timon/plugins/timon_httpsrv.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+# #############################################################################
+# Copyright : (C) 2023 by MHComm. All rights reserved
+#
+# __author__ = "Quentin Laymajoux"
+# __email__ = "info@mhcomm.fr"
+#
+# Name       : timon.plugins.timon_socksrv
+"""
```

Comment:
doc string doesn't match file name and I think it is now http and no nore a plain socket, right?

Reply:
done